workspaces/config/lib/index.js

```diff
@@ -704,6 +703,7 @@ class Config {
           continue
         }
 
+        const mapWorkspaces = require('@npmcli/map-workspaces')
```

Comment:
`loadLocalPrefix` has long since finished by the time npm is ready to reify, this should have no bearing on a global npm install.